packages/v4/src/content/design-guidelines/components/date-picker/date-picker.md

```diff
@@ -0,0 +1,143 @@
+---
+id: Date picker
+section: components
+related: ["Calendar month", "Time picker"]
+---
+
+A **date picker** helps users select a specific date, time, or range of time-related values.
+
+## Elements
+
+<img src="./img/date-picker.png" alt="Date picker elements" width="382"/>
+
+1. **Date fields**: Displays a user’s selected date. Always fill the date field with a placeholder date format (MM/DD/YYYY) or a pre-selected date before users make their selection via calendar or keyboard. The date field should never be empty.
+2. **Calendar icon**: Opens a calendar upon selection.
+3. **Selection arrows**: Allow users to navigate between months.
+4. **Calendar**: Allows users to select and navigate between days, months, years. Restricts selections by disabling certain dates.
+5. **Selected date**
+6. **Current date (today’s date)**: Indicates the day users access the calendar. Always highlighted.
+7. **Hovered state**
```

Comment:
@mcarrano would you say "hovered state" or "hover state"?

Reply:
"Hover state" is correct, I think.